client/client_pool/src/concord_client_pool.cpp

```diff
@@ -43,6 +43,14 @@ SubmitResult ConcordClientPool::SendRequest(std::vector<uint8_t> &&request,
                                             std::string correlation_id,
                                             std::string span_context,
                                             const bftEngine::RequestCallBack &callback) {
+  if (callback && timeout_ms.count() == 0) {
+    callback(bftEngine::SendResult{SubmitResult::Invalid_Argument});
+    return SubmitResult::Overloaded;
```

Comment:
We send "invalid argument" to the callback but return with "overloaded"? Which one is it?
Same below.

Reply:
As DA currently does not know how to deal with other statuses, we can only return Acknowledge/Overloaded/internal error.